packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.component.js

```diff
@@ -296,7 +296,11 @@ export class ProductAttributeValue extends PureComponent {
               block="ProductAttributeValue"
               elem="Label"
             >
-                { value }
+                { typeof value === 'string' ? (
+                    <Html content={ value } />
+                ) : (
+                    value
+                ) }
```

Comment:
In order to improve readability and extensibility, please move this to a separate method and use conditional blocks instead of the inline ternary operator.